beets/importer.py

```diff
@@ -672,7 +673,7 @@ def manipulate_files(self, move=False, copy=False, write=False,
                     # old paths.
                     item.move(copy, link)
 
-            if write and self.apply:
+            if write and (self.apply or self.choice_flag == action.RETAG):
```

Comment:
I'm a bit uncertain about what you mean: modifying the `task.apply()` method so it returns True for both `self.choice_flag in (action.APPLY, action.RETAG)`? I wouldn't oppose to that or have a strong opinion (didn't go through the implications yet), as I merely opted for the most explicit and simple way possible of adding the new action.